memory-game.css

```diff
@@ -3,45 +3,93 @@
   padding: 0;
   box-sizing: border-box;
 }
+html {
+  min-height: 100vh;
+}
 body {
   display: flex;
   flex-direction: column;
   justify-content: center;
   align-items: center;
-  height: 100vh;
   padding: 25px;
 }
 header {
-  margin-bottom: 50px;
+  width: 100vh;
+  margin-bottom: 20px;
   display: flex;
+  justify-content: space-between;
+  font-size: 1.2rem;
+  font-weight: 600;
 }
-#animals-btn,
-#weather-btn {
-  border-radius: 15px;
-  width: 45%;
-  padding: 10px;
-  margin: 5px auto;
-  background: #88bae9;
-  cursor: pointer;
-  border: none;
+
+.overlay-text {
+  display: none;
+  width: 90%;
+  max-width: 600px;
+  height: 60vh;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  flex-direction: column;
+  justify-items: center;
+  align-items: center;
+  z-index: 100;
+  padding: 30px;
+  border-radius: 10px;
+  color: white;
+  font-size: 1.5em;
```

Comment:
Good job for using rem/em :+1: 

If you have the time, I suggest practicing with rem for padding, margins etc as well.

Reply:
Thank you, I'll work on it